wallet/node_forward.go

```diff
@@ -74,9 +95,16 @@ func (n *nodeForward) Send(ctx context.Context, tx *SignedBundle, ty api.SubmitT
 	)
 }
 
-func (r *nodeForward) nextClt() api.TradingServiceClient {
-	n := atomic.AddUint64(&r.next, 1)
-	r.log.Info("sending transaction to vega node",
-		zap.String("host", r.nodeCfgs.Hosts[(int(n)-1)%len(r.clts)]))
-	return r.clts[(int(n)-1)%len(r.clts)]
+func (n *nodeForward) nextClt() api.TradingServiceClient {
+	i := atomic.AddUint64(&n.next, 1)
+	n.log.Info("sending transaction to vega node",
```

Comment:
Capital V for Vega

Reply:
well these are logs, I tend to keep them non capitalized

Reply:
So if it's not a big deal I would keep them like so.